From 5187b7a0eb6a9834e36ad8eb21109ff343e9d23c Mon Sep 17 00:00:00 2001
From: Swaroop Balan <swaroop.balan@intel.com>
Date: Thu, 4 Oct 2018 14:21:27 +0530
Subject: [PATCH] [CELADON] f2fs filesystem support enabled

f2fs filesystem support is enabled for using in adoptable storage
as per google AOSP CDD adoptable storage shall use both ext4 and f2fs. so enabling f2fs

Tracked-On: None
Test: mount cmd
      1. create a storage volume with mkfs.f2fs utility
      2. mount -t f2fs <dev entry> <mount point>
      3. check mount cmd output and check for mount type as f2fs

Signed-off-by: shyjumon <shyjumon.n@intel.com>
---
 kernel_config/kernel_64_defconfig | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/kernel_config/kernel_64_defconfig b/kernel_config/kernel_64_defconfig
index d20506b..a2d7be1 100644
--- a/kernel_config/kernel_64_defconfig
+++ b/kernel_config/kernel_64_defconfig
@@ -6887,7 +6887,11 @@ CONFIG_BTRFS_FS=m
 # CONFIG_BTRFS_DEBUG is not set
 # CONFIG_BTRFS_ASSERT is not set
 # CONFIG_NILFS2_FS is not set
-# CONFIG_F2FS_FS is not set
+CONFIG_F2FS_FS=y
+CONFIG_F2FS_STAT_FS=y
+CONFIG_F2FS_FS_XATTR=y
+CONFIG_F2FS_FS_POSIX_ACL=y
+CONFIG_F2FS_FS_SECURITY=y
 # CONFIG_FS_DAX is not set
 CONFIG_FS_POSIX_ACL=y
 CONFIG_EXPORTFS=m
-- 
1.9.1

